master/internal/resourcemanagers/resource_pool.go

```diff
@@ -410,7 +415,15 @@ func (rp *ResourcePool) receiveJobQueueMsg(ctx *actor.Context) error {
 	case job.GetJobQ:
 		ctx.Respond(rp.scheduler.JobQInfo(rp))
 	case job.MoveJob:
-		ctx.Respond(rp.moveJob(msg))
+		response, err := rp.moveJob(msg)
+		ctx.Respond(err)
```

Comment:
if the ask'er ever didn't expect a response, this panics (why `ctx.ExpectingResponse()` exists)

Reply:
The asker (job actor) should be performing an ask and checks for an error